oggm/utils/_downloads.py

```diff
@@ -341,7 +342,7 @@ def _requests_urlretrieve(url, path, reporthook, auth=None):
             raise HttpContentTooShortError()
 
 
-def _classic_urlretrieve(url, path, reporthook, auth=None):
+def _classic_urlretrieve(url, cache_path, reporthook, auth=None):
```

Comment:
I just fixed this in the opposite way myself, so this change here needs undone or it will introduce the same issue again, just reversed.